xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/VaapiEGL.cpp

```diff
@@ -113,16 +113,13 @@ bool CVaapiTexture::Map(CVaapiRenderPicture *pic)
         return false;
       }
 
-      GLint format;
-
       glGenTextures(1, &m_textureY);
       glBindTexture(m_interop.textureTarget, m_textureY);
       glTexParameteri(m_interop.textureTarget, GL_TEXTURE_MIN_FILTER, GL_LINEAR);
       glTexParameteri(m_interop.textureTarget, GL_TEXTURE_MAG_FILTER, GL_LINEAR);
       glTexParameteri(m_interop.textureTarget, GL_TEXTURE_WRAP_S, GL_CLAMP_TO_EDGE);
       glTexParameteri(m_interop.textureTarget, GL_TEXTURE_WRAP_T, GL_CLAMP_TO_EDGE);
       m_interop.glEGLImageTargetTexture2DOES(m_interop.textureTarget, m_glSurface.eglImageY);
-      glGetIntegerv(GL_IMPLEMENTATION_COLOR_READ_FORMAT, &format);
```

Comment:
keep this, I use this from time to time

Reply:
Not sure I follow - you use it for debugging? Because the variable clearly isn't used at the moment

Reply:
it's there because drivers had issues. currently I only checked in debugger but better would be a check in code if the returned format is correct.

Reply:
Guess I won't touch it for now, but it could definitely use some clean-up